src/io/flutter/view/FlutterView.java

```diff
@@ -199,18 +200,12 @@ private void addBrowserInspectorViewContent(FlutterApp app,
                                               boolean isEmbedded) {
     final ContentManager contentManager = toolWindow.getContentManager();
 
-    final String tabName;
     final FlutterDevice device = app.device();
-    if (device == null) {
-      tabName = app.getProject().getName();
-    }
-    else {
-      final List<FlutterDevice> existingDevices = new ArrayList<>();
-      for (FlutterApp otherApp : perAppViewState.keySet()) {
-        existingDevices.add(otherApp.device());
-      }
-      tabName = device.getUniqueName(existingDevices);
+    final List<FlutterDevice> existingDevices = new ArrayList<>();
+    for (FlutterApp otherApp : perAppViewState.keySet()) {
```

Comment:
I'm seeing a null pointer exception here when debugging the test that failed, it looks like during the test `perAppViewState` is null. Is this likely to only happen in the test environment? Maybe we could put this retrieving of a unique name into a helper function.

Reply:
I think that the `device` should be non-null for all normal invocations (normal app runs) but can be null for tests. I'll likely back out removing the null checks - it's not a critical back of this PR.